### ISSUE TITLE
Draft screener questions for IPP Pilot Usability Test Veterans

### ISSUE THREAD

**Issue by SophiaPhilipMO** [department-of-veterans-affairs/va.gov-team#67547]

**Comment by SophiaPhilipMO** [comment]
Screener Questions
Login Credentials; Have you ever created a Login.gov account?
Yes
No
 Do you currently use it?
Yes
No
Do you currently use any of the following credentials, select all that apply?
ID.me
My HealtheVet
DS Logon

Cognitive Considerations; Have you been diagnosed by a medical professional in the past with a diagnosis like TBI, PTSD, etc.?

Screen reader; Do you use a screen reader? If so, what type of screen reader do you use? On a scale of 1-5, with 1 being a beginner and 5 being an expert, how experienced are you with using a screen reader? [1, 2, 3, 4, 5]

**Comment by SophiaPhilipMO** [comment]
@lmorris3 @porta-antiporta check out screener questions for Veterans in the usability test

**Comment by lmorris3** [comment]
@SophiaPhilipMO May I ask the goal of the screener questions? Are they just to determine whether or not the person is a good candidate for the usability tests? And how will they be distributed? Via online survey? Are there any questions that are immediate eliminators? If so, will the survey stop if the person is "eliminated"?

**Comment by SophiaPhilipMO** [comment]
@lmorris3 I have also tagged you in the criteria that we drafted that connects to the screener questions. See [here](https://github.com/department-of-veterans-affairs/va.gov-research-repository/blob/master/research-proposals/example-product-name-1/YYYY-MM-DD-research-title/research%20plan%20template.md) for guidelines on criteria and screening questions from OCTO. We have to set up a kick off meeting with Perigean for them to review for Veterans as well. For Veterans, Perigean will use those screener questions to identify possible candidates. For proofing agents, we will need to find a recruitment method and work with site stakeholders to identify how we will get these screening questions out to identify proofing agent candidates. Yes, the survey will stop if the candidate does not fulfill the requirements.
